kotlinpoet/src/main/java/com/squareup/kotlinpoet/KOperator.kt

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2019 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.squareup.kotlinpoet
+
+enum class KOperator(
+  internal val operator: String,
+  internal val functionName: String
+) {
+    UNARY_PLUS("+", "unaryPlus"),
+    PLUS("+", "plus"),
+    UNARY_MINUS("-", "unaryMinus"),
+    MINUS("-", "minus"),
+    TIMES("*", "times"),
+    DIV("/", "div"),
+    REM("%", "rem"),
+    PLUS_ASSIGN("+=", "plusAssign"),
+    MINUS_ASSIGN("-=", "minusAssign"),
+    TIMES_ASSIGN("*=", "timesAssign"),
+    DIV_ASSIGN("/=", "divAssign"),
+    REM_ASSIGN("%=", "remAssign"),
+    INC("++", "inc"),
+    DEC("--", "dec"),
+    EQUALS("==", "equals"),
+    NOT_EQUALS("!=", "equals"),
+    NOT("!", "not"),
+    RANGE_TO("..", "rangeTo"),
+    CONTAINS("in", "contains"),
+    NOT_CONTAINS("!in", "contains"),
+    GT(">", "compareTo"),
+    LT("<", "compareTo"),
+    GE(">=", "compareTo"),
+    LE("<=", "compareTo"),
+    INVOKE("", "invoke"),
+    ITERATE("in", "iterator"),
+    GET("", "get"),
```

Comment:
Should we omit get and set, since there seems to be no good way to use them with `MemberName`?

Reply:
Yes. But I think we need at least one way to import `get` or `set` operator.
It may not be the best way, but it is a way.

Reply:
How would these operators be used, can you provide an example?
Besides, you can still import members using `FileSpec.Builder.addImport()`, so we're not introducing functionality that has been missing before.

Reply:
```kotlin
addStatement("taco%M[1] = meat", set)
addStatement("taco[1]%M = meat", set)
```
They are both generate 
```Kotlin
import com.squareup.tacos.set

taco[1] = meat
```

%M just for add import

Reply:
Honestly I don't think this is good style, I don't like having no-op `%M`s as I think they'll be confusing. My vote is to remove them.

Reply:
OK, I will follow you.